packages/components/src/components/gradient-background-wallet-icon/gradient-background-wallet-icon.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { CSSProperties } from 'react';
+import './gradient-background-wallet-icon.scss';
+import classNames from 'classnames';
+
+type TGradientBackgroundWalletIcon = {
+    color: CSSProperties['backgroundColor'];
+    primary: CSSProperties['background'];
+    secondary: CSSProperties['background'];
+    size: string;
+};
+
+const GradientBackgroundWalletIcon: React.FC<React.PropsWithChildren<TGradientBackgroundWalletIcon>> = ({
+    children,
+    color,
+    primary,
+    secondary,
+    size,
+}) => (
+    <div
+        className={classNames('gradient-background-wallet-icon', {
+            [`gradient-background-wallet-icon--${size}`]: size,
+        })}
+        style={{ backgroundColor: color }}
+    >
+        <div className='gradient-background-wallet-icon__primary' style={{ background: primary }} />
+        <div className='gradient-background-wallet-icon__secondary' style={{ background: secondary }} />
+        {children && <React.Fragment>{children}</React.Fragment>}
```

Comment:
@sergei-deriv I get your point but I was following the component made by @farzin-deriv as my reference which has something similar to this.

Reply:
@aum-deriv Farzin did this check because he added `div` for `children`, in this case you just put children inside of `React.Fragment`. You have to type just `{children}` here